src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -385,46 +375,6 @@ public String getStringAssignment(
     }
   }
 
-  /**
```

Comment:
breaking change: dropping the Jackson-typed assignment methods.
This are re-implemented in the java-server SDK, so it's not a breaking change to consumers of that SDK